docs/software/communication/openmpi.md

```diff
@@ -6,5 +6,39 @@ However, [OpenMPI](https://www.open-mpi.org/) can be used as an alternative in s
 
 To use OpenMPI on Alps, it must be built against [libfabric][ref-communication-libfabric] with support for the [Slingshot 11 network][ref-alps-hsn].
 
+## Using OpenMPI
+
+!!! warning
+    Building and using OpenMPI on Alps is still [work in progress](https://eth-cscs.github.io/cray-network-stack/).
+    The instructions found on this page may be inaccurate, but are a good starting point to using OpenMPI on Alps.
+
+!!! todo
+    Deploy experimental uenv.
```

Comment:
@biddisco, @bcumming I think it'd be good if we could deploy the not fully tested uenv from @biddisco as `prgenv-gnu-ompi/25.4:alpha1` or something like that to have something stable to pull from jfrog. What do you think?

Reply:
Will do - we don't need to do this before we deploy these docs